src/latexify/transformers/identifier_replacer.py

```diff
@@ -55,21 +55,25 @@ def visit_FunctionDef(self, node: ast.FunctionDef) -> ast.FunctionDef:
                 kwonlyargs=self._replace_args(visited.args.kwonlyargs),
                 kw_defaults=visited.args.kw_defaults,
                 defaults=visited.args.defaults,
-            )
+            )  # type: ignore
```

Comment:
Could you resolve typing errors?
If it is difficult, could you use `#type: ignore[code]` rather than the wildcard `#type: ignore` to show which error code we ignored?

Reply:
I removed the python 3.8 conditions